docs/INSTALL.md

```diff
@@ -43,12 +43,12 @@ Configure the repository for an opensource build:
 bash ./configure.sh
 ```
 
-For _Windows 10_: Use WSL to run `./configure.sh`, or, alternatively, run the following command from the
-[Visual Studio Developer Command Prompt](https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022)
-and follow instructions:
+For _Windows 10_: Use WSL(make sure [g++ is installed](https://askubuntu.com/a/915751)) to run `./configure.sh`, or, alternatively, install the [Visual Studio Developer Command Prompt](https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022) (make sure to choose the "Desktop development with C++" as your workload while installing Visual Studio), check whether [cl.exe](https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170) is in the path by typing `cl` in Developer Command Prompt for VS. Run the following command and follow instructions:
```

Comment:
Let's describe separately two setups:
1. With WSL + installed g++ (sudo apt install g++), no Developer command prompt is needed.
2. Without WSL, but with Visual Studio Installer with MSVC build tools + (likely) Windows SDK (I assume that only Windows 11 may need it, Windows 10 likely will work without it), and with the Developer command prompt.

Reply:
I have defined two setups, you may review it.